content/en/blog/_posts/2023-10-18-gateway-api-time-to-switch-i2gw/index.md

```diff
@@ -0,0 +1,111 @@
+Route resources define protocol-specific rules for mapping requests from a Gateway to Kubernetes Services.
+HTTPRoute is for multiplexing HTTP or terminated HTTPS connections.
+It's intended for use in cases where you want to inspect the HTTP stream and use HTTP request data for either routing or modification, for example using HTTP Headers for routing, or modifying them in-flight.
+
+![The resources of Gateway API](gateway-api-resources.svg)
+
+### Portability
+Gateway API is designed to be more portable across different implementations, clusters and environments. It helps reduce Ingress' reliance on non-portable, provider-specific annotations, making your configurations more consistent and easier to manage across multiple clusters.
+
+### Expressiveness
+The Gateway API sets Kuberentes standard support of a wide range of features, such as header-based matching, traffic splitting, weight-based routing, request mirroring and more. With Ingress, these features need custom provider-specific annotations.
+
+### Extensibility
+Gateway API allows for custom resources to be linked at various layers of the API. This makes granular customization possible at the appropriate places within the API structure.
+
+## Getting Started
+
+### Gateway API Quick Start
+
+The first thing you will need is a Kubernetes cluster. You can set up a new cluster using [KinD](https://kind.sigs.k8s.io/) with the following command:
+```bash
+kind create cluster --name introducing-i2gw
+```
+When you create a service of type LoadBalancer in a Kubernetes cluster, typically, cloud providers (like AWS, GCP, Azure) automatically provision a cloud load balancer and allocate a public IP address for it. However, since KinD runs locally, there's no cloud provider available to automatically provision an external IP for such services.
+
+This is where MetalLB comes in. MetalLB provides a network load-balancer implementation for Kubernetes clusters that do not have external load-balancers available, like KinD.
+You'll need to [install](https://metallb.universe.tf/installation/) it in order to continue with this guide.
+
+Now you should have a working environment to start experimenting with Gateway-API. There are many great guides you can follow at https://gateway-api.sigs.k8s.io/guides/.
+
+### Migrating from Ingress to Gateway API using Ingress2Gateway
+Migrating from Ingress to the Gateway API may seem intimidating, but luckily Kubernetes SIG-Network provides a tool to simplify the process. Ingress2Gateway assists in the migration by converting your existing Ingress resources into Gateway API resources. To get started with ingress2gateway, you need to first install the tool.
+
+```
+go install github.com/kubernetes-sigs/ingress2gateway@latest
+```
+This will install ingress2gateway to $(go env GOPATH)/bin/ingress2gateway.
+
+Alternatively, you can build it from source using https://github.com/kubernetes-sigs/ingress2gateway#build-from-source.
```

Comment:
```suggestion
Alternatively, you can build it from source by following [Build from source](https://github.com/kubernetes-sigs/ingress2gateway/tree/d277aad7a2f55519f15a9bf99f5b733113702fda#build-from-source).
```

For a blog article, a permalink to a ref is better. Folks can update that later if they want to but at least it won't end up as a broken link.

Reply:
changed, thanks